frontend/pages/projects/_id/affective-annotation/index.vue

```diff
@@ -574,16 +528,9 @@ export default {
       'getQuestionnaire'
     ]),
 
-    onScrollListener: _.debounce(function () {
-      if (this.isLoaded) {
-        const entityEditor = this.$refs.entityEditor
-        const margin = 100
-        if (entityEditor) {
-          const { bottom } = entityEditor.$el.getBoundingClientRect()
-          this.hasStickyView = bottom - margin < 0
-        }
-      }
-    }, 50),
+    toggleProgressBar() {
+      this.showProgressBar = !this.showProgressBar
+    },
 
     getDimensionComponentRef() {
```

Comment:
Since you are working at it, can you remove this function as well?

Reply:
I deeply apologize, I didn't realize the fix for the mobile toolbar was included in the refactor PR. 😓 I just found it while testing the change in different screen sizes and I thought to just include it since the fix is rather small....

You mean the `getDimensionComponentRef` function, right? Ok i will remove it here